cli/pcluster/resources/batch/docker/scripts/entrypoint.sh

```diff
@@ -3,12 +3,20 @@ set -e
 
 echo "Starting Job ${AWS_BATCH_JOB_ID}"
 
-# mount nfs
-/parallelcluster/bin/mount_nfs.sh "${MASTER_IP}" "${SHARED_DIR}"
+ebs_shared_dirs=$(echo ${SHARED_DIR} | tr "," " ")
```

Comment:
quote "${SHARED_DIR}"

Also would it be worth renaming SHARED_DIR to SHARED_DIRS?